src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -102,14 +107,23 @@ public void setUiListDelivery() {
                 .collect(Collectors.toCollection(FXCollections::observableArrayList));
     }
 
+    /**
+     * Transforms the given delivery to a ListItem to be shown in the UI List.
+     *
+     * @param delivery the delivery to be transformed.
+     * @return the ListItem to be shown in the UI List.
+     */
     private ListItem transformDeliveryToListItem(Delivery delivery) {
-        return new ListItem(String.format("[%d] %s", delivery.getDeliveryId(), delivery.getName()),
-                String.format("Ordered on: %s", delivery.getOrderDate().toString()),
-                delivery.getStatus().toString(),
-                String.format("Deliver by: %s", delivery.getDeliveryDate().toString()));
+        String title = String.format("[%d] %s", delivery.getDeliveryId(), delivery.getName());
+        String description = String.format("Customer: %s", delivery.getCustomer().getName());
```

Comment:
Should we try to abide by LoD for this i.e., implement a `getCustomerName()` from `Delivery.java`?